### PR TITLE
Refactor RestAppPage formatting (#7033) - Improved  method with prope…

### PR DESCRIPTION
**Pull request by jayasekharreddy-telluri**
…r line breaks and indentation

### CODE REVIEW

File: src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache [hunk 1]
```diff
@@ -30,18 +30,19 @@ public class Rest{{baseName}}Page<T> {
     hasNext = builder.hasNext;
   }
 
-  public static <S, T> Rest{{baseName}}Page<T> from({{baseName}}Page<S> source, Function<S, T> mapper) {
-    Assert.notNull("source", source);
-    Assert.notNull("mapper", mapper);
-      return new Rest{{baseName}}PageBuilder<>(source.content().stream().map(mapper).toList())
+public static <S, T> Rest{{baseName}}Page<T> from({{baseName}}Page<S> source, Function<S, T> mapper) {
```
Comment (DamnClin) [inline src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:33]: still need a 2 spaces indentation for the whole methode

**Review by DamnClin — CHANGES_REQUESTED** [review]
You also need to rewrite your commit message

File: src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache [hunk 2]
```diff
@@ -30,18 +30,20 @@ public class Rest{{baseName}}Page<T> {
...
+
+  Assert.notNull("source", source);
```
Comment (pascalgrimaud) [inline src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:35]: it is still not perfect, as you broke the format:

image

Comment (jayasekharreddy-telluri) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:35]: image
is the above thing is fine?

Comment (pascalgrimaud) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:35]: no, it's still wrong. You can follow the original ticket, the modification is really simple:
- see https://github.com/jhipster/jhipster-lite/blob/main/src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache#L35-L36
- add a empty line between l35 and l36
- add correct space at l36

image

In your PR, you modify the line 33 which is correct
Same modification for line 44, which is already correct

Comment (jayasekharreddy-telluri) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:35]: image

 hi @pascalgrimaud  is that fine now?

File: src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache [hunk 3]
```diff
@@ -30,18 +30,20 @@ public class Rest{{baseName}}Page<T> {
...
+  return new Rest{{baseName}}PageBuilder<>(source.content().stream().map(mapper).toList())
       .currentPage(source.currentPage())
       .pageSize(source.pageSize())
       .totalElementsCount(source.totalElementsCount())
       .pageCount(source.pageCount())
       .hasPrevious(source.hasPrevious())
       .hasNext(source.hasNext())
       .build();
-  }
+}
```
Comment (DamnClin) [inline src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:46]: This closing bracket needs the two spaces :)

File: src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache (hunk 2 above)
Comment (pascalgrimaud) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:35]: Still not. See my comment above :-)

File: src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache (hunk 3 above)
Comment (jayasekharreddy-telluri) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:46]: image

i hope now its ok 🙄?

Comment (DamnClin) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache:46]: The screenshot is ok but you need to push your code

Comment (pascalgrimaud) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache]: L33, 34, 35, 37 are still wrong for me

Comment (jayasekharreddy-telluri) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache]: image
 
now its correct?

Comment (pascalgrimaud) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache]: Yes I think :-)

**Review by pascalgrimaud — APPROVED** [review]

File: src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache (hunk 3 above)
Comment (jayasekharreddy-telluri) [reply src/main/resources/generator/server/pagination/rest/main/RestAppPage.java.mustache]: finally completed😊

**Pull request merged**